### PR TITLE
Fix discuss links and badges

### PR DESCRIPTION
**Pull request by julianoes**
The forums now has https. Moreover, the subforum should be renamed from DroneCore to Dronecode.

### CODE REVIEW

File: en/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Dronecode SDK
 
-[![Slack](https://px4-slack.herokuapp.com/badge.svg)](http://slack.px4.io)&nbsp;[![Discuss](https://img.shields.io/badge/discuss-DroneCore-ff69b4.svg)](http://discuss.px4.io/c/dronecore) [![jenkins build status](http://ci.px4.io:8080/buildStatus/icon?job=Dronecode/DronecodeSDK/develop)](http://ci.px4.io:8080/blue/organizations/jenkins/Dronecode%2FDronecodeSDK/branches/)
+[![Slack](https://px4-slack.herokuapp.com/badge.svg)](http://slack.px4.io)&nbsp;[![Discuss](https://img.shields.io/badge/discuss-Dronecode-ff69b4.svg)](https://discuss.px4.io/c/dronecode) [![jenkins build status](http://ci.px4.io:8080/buildStatus/icon?job=Dronecode/DronecodeSDK/develop)](http://ci.px4.io:8080/blue/organizations/jenkins/Dronecode%2FDronecodeSDK/branches/)
```
Comment (hamishwillee) [inline en/README.md]: This link seems wrong: https://discuss.px4.io/c/dronecode

I agree dronecore is not right, but isn't dronecode for discussions about dronecode - not the SDK?

File: en/README.md [hunk 2]
```diff
@@ -33,20 +33,20 @@ The Dronecode SDK is in beta development.
...
+
 
 ## Getting Help
 
-This guide contains information and examples showing how to use the SDK. 
+This guide contains information and examples showing how to use the SDK.
 If you have specific questions that are not answered by the documentation, these can be raised on:
 
-* [Discuss board](http://discuss.px4.io/c/sdk)
+* [Discuss board](https://discuss.px4.io/c/dronecode)
```
Comment (hamishwillee) [inline en/README.md]: As above, original link seems better.

File: en/cpp/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <div style="float:right; padding:10px; margin-right:20px;"><a href="https://www.dronecode.org/sdk/"><img src="../../assets/site/sdk_logo_full.jpg" title="Dronecode SDK Logo" width="400px"/></a></div>
 # Dronecode SDK C++ Library
-[![Slack](https://px4-slack.herokuapp.com/badge.svg)](http://slack.px4.io)&nbsp;[![Discuss](https://img.shields.io/badge/discuss-DroneCore-ff69b4.svg)](http://discuss.px4.io/c/dronecore) [![jenkins build status](http://ci.px4.io:8080/buildStatus/icon?job=Dronecode/DronecodeSDK/develop)](http://ci.px4.io:8080/blue/organizations/jenkins/Dronecode%2FDronecodeSDK/branches/)
+[![Slack](https://px4-slack.herokuapp.com/badge.svg)](http://slack.px4.io)&nbsp;[![Discuss](https://img.shields.io/badge/discuss-Dronecode-ff69b4.svg)](https://discuss.px4.io/c/dronecode) [![jenkins build status](http://ci.px4.io:8080/buildStatus/icon?job=Dronecode/DronecodeSDK/develop)](http://ci.px4.io:8080/blue/organizations/jenkins/Dronecode%2FDronecodeSDK/branches/)
```
Comment (hamishwillee) [inline en/cpp/README.md]: As above - link to discuss not good. I assume link for badge is fine.

File: en/README.md (hunk 1 above)
Comment (julianoes) [reply en/README.md]: Good  point.

Comment (julianoes) [reply en/README.md]: And the subforum is actually called "Dronecode SDK" and the URL is `/sdk`, I just got that all wrong!

**Comment by hamishwillee** [comment]
Looks good to me! Thanks. Merged.

**Pull request merged**